src/modelfree/configs/multi/train.py

```diff
@@ -15,6 +16,90 @@
 TARGET_VICTIM = collections.defaultdict(lambda: 1)
 TARGET_VICTIM['multicomp/KickAndDefend-v0'] = 2
 
+HYPERPARAM_SEARCH_VALUES = {
+    'seed': tune.sample_from(
+        lambda spec: np.random.randint(1000)),
+
+    # Dec 2018 experiments used 2^11 = 2048 batch size.
+    # Aurick Zhou used 2^14 = 16384; Bansal et al use 409600 ~= 2^19.
+    'batch_size': tune.sample_from(
+        lambda spec: 2 ** np.random.randint(11, 16)),
+
+    'rl_args': {
+                # PPO2 default is 0.01. run_humanoid.py uses 0.00.
+                'ent_coef': tune.sample_from(
+                    lambda spec: np.random.uniform(low=0.00, high=0.02)),
+
+                # nminibatches must be a factor of batch size; OK provided power of two
+                # PPO2 default is 2^2 = 4; run_humanoid.py is 2^5 = 32
+                'nminibatches': tune.sample_from(
+                    lambda spec: 2 ** (np.random.randint(0, 7))),
+
+                # PPO2 default is 4; run_humanoid.py is 10
+                'noptepochs': tune.sample_from(
+                    lambda spec: np.random.randint(1, 11)),
+                },
+    # PPO2 default is 3e-4; run_humanoid uses 1e-4;
+    # Bansal et al use 1e-2 (but with huge batch size).
+    # Sample log-uniform between 1e-2 and 1e-5.
+    'learning_rate': tune.sample_from(
+        lambda spec: 10 ** (-2 + -3 * np.random.random())),
+}
+
+MULTI_TRAIN_LOCATION = "/home/ubuntu/aws_private/multi_train"
```

Comment:
Hardcoding this is sad -- it'll break on local machines.

Maybe we should introduce a helper method, `get_data_location()`, that can default to just `data/` (relative location) but will look in an environment variable `ADV_DATA_ROOT` that we can then set in the Ray config to `/home/ubuntu/aws_private`.

Reply:
Sorry, I was just trying to get something that worked initially without initially adding a lot of complexity. I'll try to get something with your ADV_DATA_ROOT suggestion working.